lib/Core/MemoryManager.cpp

```diff
@@ -44,7 +44,7 @@ llvm::cl::opt<unsigned> RedZoneSpace(
                    "important to detect out-of-bound accesses (default=10)."),
     llvm::cl::init(10));
 
-llvm::cl::opt<uint64_t> DeterministicStartAddress(
+llvm::cl::opt<unsigned long long> DeterministicStartAddress(
```

Comment:
The problem is with uint64, it's not supported by the default template of the opt command. Therefore KLEE didn't accept an address at all. Changing it to unsigned long long fixed the problem.